documentation/modules/upgrading-mtv-ui.adoc

```diff
@@ -15,6 +15,8 @@ You must upgrade to the next release without skipping a release, for example, fr
 
 .Procedure
 
+. In the {ocp-short} web console, click *Operators*->*Installed Operators*->*{operator-name}*->*Subscription*.
```

Comment:
@stoobie Hi,
1. Regaring {operator-name}  - shouldn't it be {operator-name-ui} to have the operator name as it is in UI?
2. only the word **Operators** is bold
![Selection_371](https://user-images.githubusercontent.com/60384172/157672508-aaf5689b-94d7-496e-9ad8-0c007af9804b.png)

Reply:
You are correct! Fixed.

Reply:
Could you please fix also point number-2 above: only the word 'Operators' is bold?

Reply:
Yes, sorry I missed that. I didn't realize I needed to add some spaces. Fixed now.